components/database/mariadb-106/Makefile

```diff
@@ -27,6 +26,7 @@ COMPONENT_NAME=		mariadb
 COMPONENT_MAJOR_VERSION=10.6
 COMPONENT_MINOR_VERSION=12
 COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
+COMPONENT_REVISION=	1
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
You don't need to set COMPONENT_REVISION as the version hasn't been published.

Reply:
I confused it with php-8_1. That was one of the failing builds after the mariadb upgrade.